tasks/test-project/test-project

```diff
@@ -323,8 +323,7 @@ const globalTasks = () =>
             getExecaOptions(OUTPUT_PROJECT_PATH)
           )
         },
-        enabled: () =>
-          (!typescript && !rebuildFixture) || (typescript && !rebuildFixture),
+        enabled: () => !typescript,
```

Comment:
```suggestion
        (!typescript && !rebuildFixture) || (typescript && !rebuildFixture),
```

rebuildFixture needs to be TS and when true needs to ignore the --ts option

Reply:
This was the reason a JS project was being generated, and type checks were failing btw!

Reply:
Ok, this looks good, too:
https://github.com/redwoodjs/redwood/blob/fec6b60c6db71f6be233b22bb9310f673d8ac4d6/.github/workflows/smoke-test.yaml#L50

What does the logic need to be so that passing --rebuildFixture:
1. ensures ts-to-js is _not_ run
2. regardless of any other options passed?

Maybe add the condition inside the block instead of at Task.enabled?

Reply:
Lets try and separate thse two things.

I will create the script to rebuild the fixture in another pr because it doesn't quite work as expected anyway e.g. have to specify a path, even though it ignores whatever I pass

Reply:
> I will create the script to rebuild the fixture in another pr because it doesn't quite work as expected anyway e.g. have to specify a path, even though it ignores whatever I pass

@dac09 please don't take this direction